packages/devtools_app/web/debugger_screen.html

```diff
@@ -0,0 +1,152 @@
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+-->
+<html>
+
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script>
+        const GA_DEVTOOLS_PROPERTY = 'UA-26406144-34'; // Dart DevTools GA Property UA.
+
+        function getDevToolsPropertyID() {
+            return GA_DEVTOOLS_PROPERTY;
+        }
+    </script>
+    <!-- The below URI ?id= must match the GA_DEVTOOLS_PROPERTY above. -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-34"></script>
+    <script type="text/javascript" src="devtools_analytics.js"></script>
+    <!-- End of DevTools Google Analytics -->
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="favicon.png" rel="icon" sizes="64x64">
+    <link href="packages/octicons_css/octicons_4.4.0.css" rel="stylesheet">
+    <link href="packages/polymer_css/flex.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/debugger/debugger.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/inspector/inspector.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/logging/logging.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/performance/performance.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/profiler/profiler.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/info/info.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/timeline/timeline.css" rel="stylesheet">
+
+    <link rel="stylesheet" href="packages/primer_css/primer_12.1.0.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles_light.css">
+    <style>
+        .container {
+            /* The Flutter app takes care of container sizing now.*/
+            margin-right: 0px;
+            margin-left: 0px;
+        }
+    </style>
+
+    <script src="packages/ansi_up/ansi_up.js"></script>
+
+    <link href="packages/codemirror/codemirror.css" rel="stylesheet">
+    <script src="packages/codemirror/codemirror.js"></script>
+
+    <script src="packages/devtools_app/src/ui/plotly.js"></script>
+    <script type="text/javascript" src="packages/plotly_js/plotly.min.js"></script>
+
+    <script type="text/javascript" src="packages/split/split.min.js"></script>
+</head>
+
+<body layout vertical style="width: 100%; height: 100%; background-color: transparent;">
+
+    <header flex wrap none class="masthead legacy-dart" id="header" layout horizontal hidden>
+        <a class="masthead-logo" id="title">
+            <span class="optional-550">Dart DevTools</span>
+        </a>
+
+        <div flex horiz-padding></div>
+
+        <nav class="masthead-nav" id="main-nav">
+            <!-- add the known pages -->
+            <a disabled>
+                <span class="octicon octicon-device-mobile"></span>
```

Comment:
why is all this content still here? it doesn't make sense if this is just the debugger screen.

Reply:
The framework crashes if it can't find this content.  I've marked it as hidden to get it to work.

Ideally, we strip down Framework and only take the steps necessary to show the debugger screen.

Reply:
better yet. rewrite the debugger page so the only dart:html dependency is the codemirror portion rather than the full page :)